src/EditorFeatures/Core/Classification/CopyPasteAndPrintingClassificationBufferTaggerProvider.Tagger.cs

```diff
@@ -90,7 +90,7 @@ public IEnumerable<ITagSpan<IClassificationTag>> GetTags(NormalizedSnapshotSpanC
         }
 
         private static IEnumerable<ITagSpan<IClassificationTag>> GetIntersectingTags(NormalizedSnapshotSpanCollection spans, TagSpanIntervalTree<IClassificationTag> cachedTags)
-            => SegmentedListPool<ITagSpan<IClassificationTag>>.ComputeList(
+            => SegmentedListPool<TagSpan<IClassificationTag>>.ComputeList(
```

Comment:
switched everything to be TagSpan based.  Avoids lots and lots and lots and lots of virtual dispatch.  Everything is non-virtual on the impl type, allowing for major inlining.